src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/customize_panel/customize_panel_editor.tsx

```diff
@@ -170,9 +176,16 @@ export const CustomizePanelEditor = (props: CustomizePanelProps) => {
               size="xs"
               data-test-subj="resetCustomEmbeddablePanelDescriptionButton"
               onClick={() => {
-                setPanelDescription(embeddable.getOutput().defaultDescription);
+                setPanelDescription(
+                  embeddable.getInput().description ?? embeddable.getOutput().defaultDescription
+                );
```

Comment:
This change should be undone: the reset button should always set it back to the default description.

```suggestion
                setPanelDescription(embeddable.getOutput().defaultDescription);
```